Zeeguu Reader/Zeeguu Reader/ArticleListViewController.swift

```diff
@@ -116,26 +116,17 @@ class ArticleListViewController: ZGTableViewController {
 	}
 
 	override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-		let queueCell = tableView.dequeueReusableCellWithIdentifier("Cell")
-		var cell: UITableViewCell
+		let queueCell = tableView.dequeueReusableCellWithIdentifier("Cell") as? ArticleTableViewCell
+		
+		let article = articles[indexPath.row]
```

Comment:
it would be cool to have a comment for the arguments of this function.
i'd like to know what's the indexPath, at least